### PR TITLE
Update Notification Docs and Add Flow Charts

### PR DESCRIPTION
**Pull request by esethna**
Flow Charts have been reviewed by dev. Adding QA to review as well.

Also cleaned up the Getting Started > Configuring Notifications page while doing so as it had a lot of out of date and unnecessary information for an onboarding end user

### CODE REVIEW

File: source/help/getting-started/configuring-notifications.rst [hunk 1]
```diff
@@ -1,124 +1,86 @@
...
-message in the channel.
-
-You can choose not to show unread indicators in a channel by clicking
-**Channel Menu** > **Notification Preferences** > **Mark Channel
-Unread** and selecting **Only for mentions**. This will only notify you if your name is mentioned or a
-keyword for which you're listening is entered.
+- Direct Messages you
+- Mentions your username or first name in a channel
+- Notifies a channel you're in using @channel or @all
+- Uses any `keywords you've configured<https://docs.mattermost.com/help/settings/account-settings.html#words-that-trigger-mentions>`_
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst]:
```suggestion
- Uses any `keywords you've configured <https://docs.mattermost.com/help/settings/account-settings.html#words-that-trigger-mentions>`_ 
```

File: source/help/getting-started/configuring-notifications.rst [hunk 2]
```diff
@@ -1,124 +1,86 @@
...
-there are unread messages in the channel. Clicking on the channel
-removes the bold indicator, and brings you to the most recent unread
-message in the channel.
-
-You can choose not to show unread indicators in a channel by clicking
-**Channel Menu** > **Notification Preferences** > **Mark Channel
-Unread** and selecting **Only for mentions**. This will only notify you if your name is mentioned or a
-keyword for which you're listening is entered.
+- Direct Messages you
+- Mentions your username or first name in a channel
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst]: "first name" depends on the Words that trigger mentions setting.

File: source/help/getting-started/configuring-notifications.rst [hunk 3]
```diff
@@ -1,124 +1,86 @@
...
+Muting a Channel
 ----------------------------------------
 
-If Mattermost is open in a browser tab, the favicon updates to inform you of unread messages and
-mentions. Browser tab notifications are available on Firefox and Chrome.
+Channels in the sidebar are **bold** when there are unread messages in the channel and show a badge count if you are mentioned specifically. To mute a channel so it does not appear unread and you do not recieve notifications, click the channel name at the top of the page to access the **Channel Menu** > **Notification Preferences > Mute channel**.
 
-- Unread messages are denoted by an asterisk (\*) next to the Mattermost icon.
-- Unread mentions are counted in brackets and incorporate mentions and direct messages from all of your teams.
+.. tip :: Mentions are triggered by `selected keywords <https://docs.mattermost.com/help/settings/account-settings.html#words-that-trigger-mentions>`_, learn more about `mentioning teammates <http://docs.mattermost.com/help/messaging/mentioning-teammates.html>`__.
```
Comment (amyblais) [inline source/help/getting-started/configuring-notifications.rst]:
```suggestion
.. tip :: Mentions are triggered by `selected keywords <https://docs.mattermost.com/help/settings/account-settings.html#words-that-trigger-mentions>`_. Learn more about `mentioning teammates <http://docs.mattermost.com/help/messaging/mentioning-teammates.html>`__.
```

File: source/help/getting-started/configuring-notifications.rst (hunk 2 above)
Comment (esethna) [reply source/help/getting-started/configuring-notifications.rst]: True, but by default this is on. I'm attempting to state the default settings that will trigger notifications which is "mentions and DMs"

**Comment by esethna** [comment]
@lindalumitchell or @lfbrock any comments on this or are we okay to merge?

File: source/overview/faq.rst [hunk 4]
```diff
@@ -69,12 +69,39 @@ To add yourself to the `Mattermost page on LinkedIn <https://www.linkedin.com/co
...
+.. image:: ../images/desktop_notification_chart.png
+  :width: 239px
+  :height: 232px
+
+What determines if an email notification should be triggered?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Email notifications are triggered under to following conditions. Click to expand the flow chart.
+
+.. image:: ../images/email_notification_chart.png
```
Comment (lindalumitchell) [inline source/overview/faq.rst]: Is "or less" accurate for the language about batching? Does the system check in regular intervals on the clock regardless of activity, or does the timer start when a notification is triggered, and then after 15 minutes (or an hour) with no further triggers the batch email is sent? I thought it was the latter, but I'm not 100% sure now.

Screen Shot 2019-10-10 at 11 46 01 AM

Comment (esethna) [reply source/overview/faq.rst]: The latter is correct, but if you have a message at T+0, then another at T+12, you'll get a notification at T+15. The "or less" is meant to indicate that if there are more messages after the first, the notification may come sooner

Comment (lindalumitchell) [reply source/overview/faq.rst]: Ah okay, I see, thanks.

**Pull request merged**